src/pybind/mgr/dashboard/frontend/src/app/core/auth/user-form/user-form.component.ts

```diff
@@ -128,10 +129,19 @@ export class UserFormComponent extends CdForm implements OnInit {
     const observables = [this.roleService.list(), this.settingsService.getStandardSettings()];
     observableForkJoin(observables).subscribe(
       (result: [UserFormRoleModel[], CdPwdExpirationSettings]) => {
-        this.allRoles = _.map(result[0], (role) => {
-          role.enabled = true;
-          return role;
-        });
+        // this.allRoles = _.map(result[0], (role) => {
+        //   role.enabled = true;
+        //   return role;
+        // });
+        // this.allRoles = result[0].map((role) => ({ content: `${role.name }, ${role.description}` , enabled: true  }));
+        this.allRoles = result[0].map((role) => ({
+          name: role.name,
+          description: role.description,
+          selected: role.selected,
+          scopes_permissions: role.scopes_permissions,
+          enabled: role.enabled,
+          content: `${role.name}, ${role.description}`
+        }));
```

Comment:
do we need this? the way i see it, below should be enough to make this work..
```
  this.allRoles = _.map(result[0], (role) => {
          role.enabled = true;
          role.content = `${role.name}, ${role.description}`
          return role;
        });
```

Reply:
suggested changes are done and code is committed